src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -109,6 +129,8 @@ const GuideStore = Reflux.createStore({
       );
     }
 
+    bestGuide ? this.updatePrevGuide(bestGuide) : null;
```

Comment:
two things - i would favor if statements over ternaries for side effects, and I think it's better to put this logic into the update method and have the "don't do this if X" check live there.